_schema.graphql

```diff
@@ -914,6 +914,7 @@ type ArtistRelatedData {
   genes(after: String, first: Int, before: String, last: Int): GeneConnection
   artists(
     exclude_artists_without_artworks: Boolean = true
+    min_forsale_artworks: Int
```

Comment:
<!--
  1 failure:  Addition uses `sn...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 917;
-->
- :no_entry_sign: Addition uses `snake_case` instead of `camelCase`: `min_forsale_artworks`